kernel/network/e1000.c

```diff
@@ -60,31 +64,126 @@ read_mac()
 }
 
 void
-rxinit()
+init_receive_descriptors()
 {
+  for(int i = 0; i < E1000_NUM_RX_DESC; i++) 
+    {
+      e1000->rx_descs[i].addr = RX_DESC_BASE + i * (8192 + 16);
```

Comment:
we could use makros here to reduce magic numbers

Reply:
yes

Reply:
I created a makro `E1000_SIZE_RX_DESC` for 8192 (even if that is a lot bigger than our struct) but I don't know what the meaning of 16 is.